src/status_im/contexts/wallet/send/send_amount/view.cljs

```diff
@@ -2,11 +2,13 @@
   (:require
     [quo.theme]
     [status-im.contexts.wallet.send.input-amount.view :as input-amount]
+    [status-im.setup.hot-reload :as hot-reload]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn view
   []
+  (hot-reload/use-safe-unmount #(rf/dispatch [:wallet/stop-get-suggested-routes]))
```

Comment:
Here we're configuring the wallet send screen to dispatch the `:wallet/stop-get-suggested-routes` event when unmounting the component. Without this, it seems that we'll continue to receive suggested route signals, which can cause some glitches because we're re-processing stale wallet send UI state.